sdk/synapse/synapse-artifacts/swagger/README.md

```diff
@@ -15,7 +15,7 @@ clear-output-folder: false
 tracing-info:
   namespace: "Azure.Synapse.Artifacts"
   packagePrefix: "Microsoft.Synapse"
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/949388b9dfddc435859fda2850c5bb5019aa217b/specification/synapse/data-plane/readme.md
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/synapse/data-plane/readme.md
```

Comment:
Would you mind adding the commit hash instead of main? In case we need to re-generate against the same swagger it makes it a lot easier.

Reply:
Updated